README.md

```diff
@@ -98,12 +98,22 @@ have set the `NODE_ENV` variable.
 
 ### Convert Preferences Data
 
-GPII has 2 set of preferences JSON5 data files:
-
-* The preferences files for running GPII are located at %gpii-universal/testData/preferences
-* The preferences files for running node tests are located at %gpii-universal/tests/data/preferences
-
-When any preferences file in either one of these 2 directories are modified, running `npm run postinstall` will generate
+GPII has two sets of preferences JSON5 data files, located at `%gpii-universal/testData/preferences` and `%gpii-universal/tests/data/preferences`:
+
+* The preferences files for running GPII and for integration tests are located at
+  `%gpii-universal/testData/preferences`.  These files are converted into two types of preferences safes and GPII keys:
+  * They are converted into `snapset` preferences safes and GPII keys and placed into the
+    `%gpii-universal/build/dbData/snapset/` folder.   These are used to update the snapsets in CouchDB when GPII is
+    run in a production or staging configuration.
+  * They are also converted into `user` preferences safes and GPII keys and placed into the
+    `%gpii-universal/build/dbData/user/` folder.   These are used with PouchDB when GPII runs in a development
```

Comment:
When GPII runs in a development configuration, it also uses snapset-type data from `%gpii-universal/build/dbData/snapset/`. See [pouchManager code](https://github.com/GPII/universal/pull/626/files#diff-ec1115f60a98f8db056cc76db4fc244aR21).

`%gpii-universal/build/dbData/user/` is only used for running integration tests. See [PouchTestCaseHolder](https://github.com/GPII/universal/pull/626/files#diff-9a09cbe02e1a616b46e9ff110b1b1452R50).

The doc in testData/dbData/README.txt for these data sets are correct. Please sync up. Thanks.